pkgs/tools/misc/vix/default.nix

```diff
@@ -13,6 +13,10 @@ stdenv.mkDerivation {
 
   nativeBuildInputs = [ autoreconfHook ];
 
+  configureFlags = [
+    (lib.enableFeature (!stdenv.isDarwin) "sdltest")
```

Comment:
this needs to be 
```suggestion
    (lib.enableFeature (!stdenv.hostPlatform.isDarwin) "sdltest")
```
now? or does it not matter.

Reply:
Oops, yeah. Old silly shorthand.